rotkehlchen/tests/unit/accounting/test_misc.py

```diff
@@ -125,8 +125,7 @@ def test_fees_count_in_cost_basis(accountant, google_service):
     )
 
     expected_pnls = PnlTotals({
-        AccountingEventType.TRADE: PNL(taxable=FVal('1550.055'), free=ZERO),
-        AccountingEventType.FEE: PNL(taxable=FVal('-300.630'), free=ZERO),
+        AccountingEventType.TRADE: PNL(taxable=FVal('319.395'), free=ZERO),
```

Comment:
Here the difference is caused by this fee https://github.com/rotki/rotki/blob/30309d115687c78a2ba0d9d713338d4f297f910b/rotkehlchen/tests/unit/accounting/test_misc.py#L104

In the previous PR status fee was always added to buy cost basis. Now I changed to take into account cases when one of the assets is fiat.

So previously `0.5` ETH * `1800` EUR /ETH = `+900` EUR to cost basis.

But since in the event that I linked ETH is sold for EUR, now it is `0.5` ETH * `1800` EUR /ETH = `-900` EUR to cost basis.

So in total there is a `1800` EUR difference in cost basis due to the fee, which caused this huge number change

Reply:
Yes, makes sense